Aga.Controls/Tree/TreeViewAdv.Draw.cs

```diff
@@ -29,6 +29,10 @@ public void AutoSizeColumn(TreeColumn column)
 						if (nc.ParentColumn == column)
 							w += nc.GetActualSize(node, _measureContext).Width;
 					}
```

Comment:
Please use tabs for indentation to stay in line with the rest of the codebase 🙂

Reply:
I was unsure how I could create a pull request using the UI of the desktop client and the easiest way I could find was to create a fork. This might not be the way it is intended. I have to read up on that.